### ISSUE TITLE
Auto Bow (Player or Combat Section)

### ISSUE THREAD

**Issue by DefyEnjoyCantCode** [ImpactDevelopment/ImpactIssues#2711]
**Is your feature request related to a problem? Please describe.**
No, really. But, this feature is likely going to be very useful for keeping your distance while still getting hits in.

**Describe the solution you'd like**
Auto Bow, is basically automated hunting, with bows. Baritone runs away from nearby players, teammates, hostile, or passive entities (changeable in it's settings), you can change the distance it must reach for it to start using the bow. (in blocks) It will pull back the bow, and shoot the target. You can also toggle a setting called Force Stop. What this does, if an entity gets too close while the bow it being pulled back, it will stop the bot from shooting. This could get annoying, so I recommend disabling it.

**Describe alternatives you've considered**
I don't really know any. No features actually do what I'm suggesting.

**Additional context**
Not really, tbh. You can change the distance it must reach before it shoots, too.

**Comment by ZWORX52** [comment]
I think this has already been mentioned - bow...... spam........... #1281 by leijurv.

**Comment by l1ving** [comment]
No, this issue mentions following and keeping your distance, though similar things have been suggested this has a lot of detail so I'm keeping it open
